routes/login.js

```diff
@@ -9,11 +10,41 @@ router.get("/", (req, res) => {
         return;
     }
     res.render('login/login');
-})
+});
 
-router.post("/", passport.authenticate('local', {failureRedirect: '/login', failureFlash: true}),(req, res) => {
-    //do passport stuff here
-    res.redirect("/class");
-})
+router.post("/", (req, res, next) => {
+    passport.authenticate('local', (err, user, info) => {
+        if (err) {
+            return next(err);
+        }
+        return db.accountLocked(req.body.username).then((lockData) => {
+            if (lockData.locked) {
+                let date = new Date(lockData.lockAccountUntil);
+                let data = {
+                    hasErrors: true,
+                    errors: [`${lockData.failedLoginAttempts} failed login attempts. Your account has been locked until ${date}`]
```

Comment:
does this error display?

Reply:
Yes. You need to fail the password about 10 times in a row in order for the time delay to be big enough for your attempt to actually be denied. At this point, you should start seeing the message.